files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc

```diff
@@ -0,0 +1,38 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [Alignment-basedReferenceView]'), '2;1');
+FILE_NAME ('ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc', '2023-02-16T14:56:46', ('Alfonso D\X2\0027\X0\Urso', 'alfonso.durso@acca.it'), ('ACCA software S.p.A.'), 'ACCA_IFC.dll Version 500', 'ACCA usBIM.infra', $);
+FILE_SCHEMA (('IFC4X4'));
+ENDSEC;
+DATA;
+#1 = IFCOWNERHISTORY(#2, #6, $, .NOCHANGE., $, $, $, 1676559406);
+#2 = IFCPERSONANDORGANIZATION(#3, #4, $);
+#3 = IFCPERSON($, 'D\X2\0027\X0\Urso', 'Alfonso', $, $, $, $, $);
+#4 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#5 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#6 = IFCAPPLICATION(#5, '200', 'ACCA usBIM.infra', 'ACCA usBIM.infra');
+#7 = IFCPROJECT('3Nm9Xuu4jBF9TR32nfXmZw', #1, 'IfcTunnel example', $, $, $, $, (#13), #8);
+#8 = IFCUNITASSIGNMENT((#9, #10, #11, #12));
+#9 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#10 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#11 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#12 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#13 = IFCGEOMETRICREPRESENTATIONCONTEXT('3D', 'Model', 3, 1.E-4, #14, #16);
+#14 = IFCAXIS2PLACEMENT3D(#15, $, $);
+#15 = IFCCARTESIANPOINT((0., 0., 0.));
+#16 = IFCDIRECTION((0., 1.));
+#17 = IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body', 'Model', *, *, *, *, #13, $, .MODEL_VIEW., $);
+#18 = IFCSITE('1ts0m9rLP4yQoNzVYotrwH', #1, 'Default Site', $, $, #19, $, $, .ELEMENT., (0, 0, 0), (0, 0, 0), 0., $, $);
+#19 = IFCLOCALPLACEMENT($, #22);
+#20 = IFCDIRECTION((0., 0., 1.));
+#21 = IFCDIRECTION((1., 0., 0.));
+#22 = IFCAXIS2PLACEMENT3D(#15, #20, #21);
+#23 = IFCRELAGGREGATES('0_td4AF1j5lweAb2kvPx$x', #1, 'Project Container', 'Project to Spatial Element', #7, (#18));
+#24 = IFCCLASSIFICATION('Default Source', 'Default Edition', '2000-01-01', 'Default Name', $, $, $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('2sTjfww6T4Y8G$LfWsT52u', #1, $, $, (#7), #24);
+#26 = IFCGROUP('1Xwt4d0Yn11B_91a8ResIh', #1, 'A group', $, $);
+#27 = IFCRELDECLARES('3R0mWNDGXDnB8Ljwb2_SSu', #1, 'Project Declaration Relationship', $, #7, (#26));
+#28 = IFCMAPCONVERSION(#13, #29, 0., 0., 0., 0., 0., 1.);
+#29 = IFCPROJECTEDCRS('EPSG:3065', 'Istituto Geografico Militare 1995 (IGM95)', 'EPSG:3065', $, 'UTM', '33N', $);

```

Comment:
Double checking: is the height-related CRS unknown?

Reply:
Fixed